cypress/support/fragments/users/loans/userLoans.js

```diff
@@ -1,6 +1,15 @@
 import { matching } from 'bigtest';
-import { Pane, MultiColumnListRow, MultiColumnListCell, HTML, including, Button } from '../../../../../interactors';
+import {
+  Pane,
+  MultiColumnListRow,
+  MultiColumnListCell,
+  HTML,
+  including,
+  Button,
+  KeyValue,
+} from '../../../../../interactors';
 import ItemVeiw from '../../inventory/inventoryItem/itemVeiw';
```

Comment:
please rename ItemVeiw -> ItemView
Also rename it in /inventory/inventoryItem/

Reply:
done